config/i18n/locales/german/meta-tags.json

```
@@ -0,0 +1,5 @@
+{
+  "title": "freeCodeCamp Programmiertutorials: Python, JavaScript, Git & mehr",
+  "keywords": "freeCodeCamp, Programmierung, Frontend, Programmierer, Artikel, reguläre Ausdrücke, Python, JavaScript, AWS, JSON, HTML, CSS, Bootstrap, React, Vue, Webpack",
+  "description": "Stöbere in Tausenden von Programmiertutorials, die von Experten geschrieben wurden. Lerne Webentwicklung, Data Science, DevOps, Software-Sicherheit und hol dir Tipps für deine Karriere als Entwickler."
+}
```

```suggestion
}

```

maybe here needs a new line character